japan/assets/attendee-prerequisites.md

```diff
@@ -26,51 +25,58 @@ GitHub の Primary email address と Linux Foundation ID のメールアドレ
-      - CNCF-<企業名が判断できる名称> (← 企業の開発者として登録している場合のみ)
-      - **Note**: もし "CNCF-<企業名が判断できる名称>" が付いているが、"Groups:Authorized CNCF Contributors" が付いてない場合は、再度手順iiから実施する。
+  + Linux Foundation ID の [Social network logins](https://identity.linuxfoundation.org/user/me/hybridauth) に
+    アクセスして GitHub アカウントと紐付けます。
+
+* CNCF CLA sign up  
+企業の開発者として登録する場合と個人の開発者として登録する場合で手順と確認項目が異なります。
+  + **企業の開発者として登録する場合**
+    1. 企業内で CNCF 開発参加者のリスト管理者に名前の追加を依頼します。
+    2. CNCFの[当該ページ](https://identity.linuxfoundation.org/projects/cncf)の "Sign up to contribute to this project as an **employee**" をクリック。(この手順で "Groups:Authorized CNCF Contributors" が設定されます。)
+    3. The Linux Foudation ID を使ってログインします。
+    4. [Linux Foundation のユーザーページ](https://identity.linuxfoundation.org/user/me) に下記が表示されることを確認します。
+        - Groups:Authorized CNCF Contributors
+        - CNCF-<企業名が判断できる名称>
+        - **Note**: もし "CNCF-<企業名が判断できる名称>" が付いているが、"Groups:Authorized CNCF Contributors" が付いてない場合は、再度「CNCF CLA sign up」のはじめから実施してください。
+  + **個人の開発者として登録する場合**
+    1. CNCFの[当該ページ](https://identity.linuxfoundation.org/projects/cncf)の "Sign up to contribute to this project as an **individual**" をクリック。(この手順で "Groups:Authorized CNCF Contributors" が設定されます。)
+    2. The Linux Foudation ID を使ってログインします。
+    3. [Linux Foundation のユーザーページ](https://identity.linuxfoundation.org/user/me) に下記が表示されることを確認します。
+        - Groups:Authorized CNCF Contributors
+        - **Note**: もし "Groups:Authorized CNCF Contributors" が付いてない場合は、再度「CNCF CLA sign up」のはじめから実施してください。
 
 # Kubernetes の Slack に参加
-* ここから参加します。https://slack.k8s.io/
+* [こちら](https://slack.k8s.io/)から参加します。
 * ログイン後に以下のチャンネルに入ってください。
   + #jp-dev
     - レビュー依頼の練習に使用します。そのほか、実際のコントリビューション (Issue や PR) のそれぞれについて議論する場です。
   + #jp-mentoring
     - 事前の質疑応答、講義中のフォローに利用します。そのほか、コミュニティとの関わり方について議論する場です。
     - 質疑やフォローでは発言に対してスレッドを作って回答していく予定です。複数の質問が混じらないようにするためご協力ください。
-    - Slackのスレッドについては [こちら](https://slack.com/intl/ja-jp/help/articles/115000769927-%E3%82%B9%E3%83%AC%E3%83%83%E3%83%89%E3%82%92%E4%BD%BF%E7%94%A8%E3%81%97%E3%81%A6%E4%BC%9A%E8%A9%B1%E3%82%92%E6%95%B4%E7%90%86%E3%81%99%E3%82%8B) を参照してください。
+    - Slackのスレッド機能については [こちら](https://slack.com/intl/ja-jp/help/articles/115000769927-%E3%82%B9%E3%83%AC%E3%83%83%E3%83%89%E3%82%92%E4%BD%BF%E7%94%A8%E3%81%97%E3%81%A6%E4%BC%9A%E8%A9%B1%E3%82%92%E6%95%B4%E7%90%86%E3%81%99%E3%82%8B)を参照してください。
 
 # コマンドのインストール
 * 以下のコマンドをインストールしておいてください。
   + git
 
 # Git や GitHub の操作
-* 基本的な Git や GitHub の操作には慣れておいてください。
-* 以下も参考にしてください。
-  + https://github.com/kubernetes/community/blob/master/contributors/guide/contributor-cheatsheet/README-ja.md#%E8%B2%A2%E7%8C%AE%E3%81%99%E3%82%8B
+* 基本的な Git や GitHub の操作に慣れておいてください。
+* [Kubernetesコントリビューターチートシートのこちらの節](https://github.com/kubernetes/community/blob/master/contributors/guide/contributor-cheatsheet/README-ja.md#%E8%B2%A2%E7%8C%AE%E3%81%99%E3%82%8B)から、GitHub での開発の概要を把握しておいてください。
+
 
 # リポジトリのフォークとクローン
 * 以下のリポジトリを自分の GitHub アカウントにフォークしておいてください。
```

Comment:
I think URL style is better than link text style for GitHub repository at line 83.